src/style.css

```diff
@@ -1 +1,55 @@
 /* Styles go here */
+body {
+  margin: 0;
+}
+
+.container {
+  margin: 0;
+  display: flex;
+}
+
+.logo {
+  margin-top: 10px;
+  margin-left: 50px;
+}
+
+.navigation {
+  width: 100vw;
```

Comment:
Dont set width:100vw (usually it is not a good practice), use flex properties instead on a **containes** element. You can simply set **justify-content: space-between** to achive this result